keyboards/illusion/rosa/info.json

```diff
@@ -20,6 +20,9 @@
     "processor": "atmega32u4",
     "bootloader": "atmel-dfu",
     "community_layouts": ["60_ansi_tsangan"],
+    "layout_aliases": {
+        "LAYOUT_all": "LAYOUT_60_ansi_tsangan_split_bs_rshift"
```

Comment:
`LAYOUT_60_ansi_tsangan_split_bs_rshift` should be named `LAYOUT_60_tsangan_hhkb` throughout this file.